packages/advi/advi.2.0.0/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+synopsis: "Active DVI Dune package!"
+description: """
+Active-DVI is a DVI previewer and presenter written in OCaml with some eye
+candy effects for presentation, support for interactive demonstrations, and
+embedding of arbitrary applications within the presentation.
+
+Active-DVI is well suited for use in combination with WhizzyTeX
+(http://cristal.inria.fr/whizzytex/)
+"""
+maintainer: ["Didier Rémy <Didier.Remy@inria.fr>"]
+authors: [
+  "Alexandre Miquel"
+  "Jun Furuse"
+  "Didier Rémy"
+  "Pierre Weis"
+  "Xavier Leroy"
+  "Roberto Di Cosmo"
+  "Didier Le Botlan"
+  "Alan Schmitt"
+]
+license: "LGPL-2.1-or-later"
+homepage: "http://advi.inria.fr"
+dev-repo: "git+https://github.com/diremy/advi.git"
+doc: "http://advi.inria.fr"
+bug-reports: "Didier.Remy@inria.fr"
+depends: [
+  "ocaml" {>= "4.11.1"}
```

Comment:
You were using the `ocaml-base-compiler` instead of the `ocaml` package. The latter includes ocaml installation in /usr/bin (via ocaml-system) and variants (flambda, …). As a general rule of thumb, only the `ocaml` package should be used.